agents-common/src/main/java/org/apache/ranger/plugin/policyevaluator/RangerCustomConditionEvaluator.java

```diff
@@ -37,7 +37,6 @@
 import java.util.Collections;
 import java.util.List;
 
-
 //
 // this class should have been named RangerConditionEvaluatorFactory
```

Comment:
nit: can be removed.

Reply:
Updated this comment to clarify that this class is a factory that instantiates `RangerConditionEvaluator` implementations.